app/src/main/java/org/greenstand/android/TreeTracker/models/StepCounter.kt

```diff
@@ -53,11 +62,4 @@ class StepCounter(
             }
         }
     }
-
-    companion object {

```

Comment:
Given there is no intention to expose the keys for accessing the values directly from preferences, I made these private val members of the class. The need isn't there to advocate for a singleton companion class.

Reply:
The idea behind companion objects is to have 'static' values. There is no need for these values to be tied to the class instance.

Reply:
Private val makes sense, but we should leave them in a companion object.

Reply:
I don't get the reason for companion object here. If it were to be accessed/referenced from outside what you said makes sense to me. Absence of that, I don't get it. Could you please point out what I am missing?

Reply:
@Elforama, I see that you want to keep them as static private variables. Sorry I missed to get it from your earlier comment. Updating it. Sometimes I miss the obvious.